stdlib/logging/__init__.pyi

```diff
@@ -326,7 +326,9 @@ class Filter:
     def filter(self, record: LogRecord) -> bool: ...
 
 class LogRecord:
-    args: _ArgsType
+    # args can be set to None by logging.handlers.QueueHandler
+    # (see https://bugs.python.org/issue44473)
+    args: Optional[_ArgsType]
```

Comment:
Nit: We prefer `| None` syntax for new and changed code:

```suggestion
    args: _ArgsType | None
```

Reply:
done